pdns/rec_channel_rec.cc

```diff
@@ -928,6 +928,27 @@ static uint64_t doGetThreadCPUMsec(int n)
   return tt.times.at(n);
 }
 
+static ProxyMappingStats_t* pleaseGetProxyMappingStats()
+{
+  auto ret = new ProxyMappingStats_t;
+  if (t_proxyMapping) {
+    for (const auto& [key, entry] : *t_proxyMapping) {
+      ret->emplace(std::make_pair(key, ProxyMappingCounts{entry.stats.netmaskMatches, entry.stats.suffixMatches}));
+    }
+  }
+  return ret;
+}
+
+static string doGetProxyMappingStats()
+{
+  ostringstream ret;
+  auto m = broadcastAccFunction<ProxyMappingStats_t>(pleaseGetProxyMappingStats);
```

Comment:
Just a nitpick, but perhaps renaming `m` to `proxyMappingStats` would be better.